main-site/components/faq-section/Data.tsx

```diff
@@ -0,0 +1,80 @@
+const data = [
```

Comment:
Let's move everything in this file to data.ts in lib! Also please add a type and name your data object to be more descriptive like faqSectionData or something!

<img width="637" alt="Screen Shot 2022-10-07 at 1 22 16 PM" src="https://user-images.githubusercontent.com/59738880/194614678-57dcdf3d-5893-44a4-b1bc-d82593354657.png">